helper/logging/logger_test.go

```diff
@@ -0,0 +1,170 @@
+package logging
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/consul/sdk/testutil"
+	log "github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLogger_SetupBasic(t *testing.T) {
+	cfg := LogConfig{LogLevel: log.Info}
+
+	logger, err := Setup(cfg, nil)
+	require.NoError(t, err)
+	require.NotNil(t, logger)
+}
+
+func TestLogger_SetupInvalidLogLevel(t *testing.T) {
+	cfg := LogConfig{LogLevel: 999}
+
+	_, err := Setup(cfg, nil)
+	testutil.RequireErrorContains(t, err, "invalid log level")
```

Comment:
I'm less concerned about imports for tests than I am for prod code, and yet: is it worth depending on consul/sdk/testutil just for this and testutil.TempDir, esp when there's now a builting testing.TempDir in newer Go versions?